src/en/experts-sla.md

```diff
@@ -1,149 +1,257 @@
-Title: Managed solutions under SLA
+Title: Managed solutions
 
-# Managed solutions under SLA
+# Managed solutions
 
-[Juju experts](https://jujucharms.com/experts) may designate a Juju
-model as a managed, supported solution by setting an SLA.
+A Juju Expert is a partner backed by professional support at Canonical Ltd.
+Such a partner deploys and supports an infrastructure based on Juju. We call
+this a *managed solution*.
 
-There are three support levels that may be set on a model:
+Any support requests beyond the build, workload, and operation of a particular
+solution can be escalated to Canonical by means of their Ubuntu Advantage
+support framework. Prime examples of such requests are for issues relating to
+Juju itself, Ubuntu-specific software, or the operating system kernel. The
+escalated support case will be treated by Canonical at the same support level
+currently ascribed to the partner. Hence, support level names used by both a
+Juju Expert and by Canonical are the same:
```

Comment:
I'd just state that there's a common set of support levels and not get into the customer vs the canonical one. Especially since they're common as part of the expert program it's like telling the user "they could be different words for this, but we've made sure both parties use the same words"

Reply:
This is fairly innovative and non-intuitive and I wanted to dispel any confusion by making it transparent.

Reply:
We don't know. They can very well be looking for a solution in both camps. It's better to be transparent so the customer will be in a better position to take a decision.